### PR TITLE
Clarify go install method

### LINKED ISSUE #629: [Site Bug] Backup/restore instructions problem [Issue with the page: /timescaledb/latest/how-to-guides/backup-and-restore/timescaledb-backup/]
_Use this template for reporting bugs in the docs._

# Describe the bug

The `go get` command for installing `timescaledb-backup` appears to be broken. The command returns with a `can't load package` error.

### Full output:
```
ubuntu@ip-x-x-x-x:~$ go get github.com/timescale/timescaledb-backup/
can't load package: package github.com/timescale/timescaledb-backup: no Go files in /home/ubuntu/go/src/github.com/timescale/timescaledb-backup
```

This error is occuring on an AWS instance running Ubuntu 20.04 LTS. 
Go installed with: `apt get install golang-go`
Version:
 ```
ubuntu@ip-x-x-x-x:~$ go version
go version go1.13.8 linux/amd64
```

## What do the docs say now?

> Install timescaledb-backup:
> ```
> $ go get github.com/timescale/timescaledb-backup/
> ```

## What should the docs say?
I don't know.

# Page affected

https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/timescaledb-backup/#installing-timescaledb-backup

# Version affected

**_latest_**

# Subject matter expert (SME)

na

# Screenshots

na

# Any further info

na

### PR DESCRIPTION
**Pull request by snspinn**
Go get can only download the source files, as it stands. There is no one-liner go command that downloads and installs, to my knowledge and testing.
This procedure should get Go beginners setup with the ts-dump and ts-restore tools. This method is only two command longer than downloading the prebuilt binaries.
Tested on Ubuntu 20.04 only.

# Description

I think the Go install method description is too short. This will guide users once the go get command either appears to do nothing or appears to error out.

# Version

Which documentation version does this PR apply to?

- [* ] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #629

### CODE REVIEW

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md [hunk 1]
```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
...
+    $ go get -d github.com/timescale/timescaledb-backup/
+    # Ignore any can't load packages errors, if any.
+    ```
+1. Build the binaries:
+   ```bash
+   $ cd ~/go/src/github.com/timescale/timescaledb-backup/
+   $ go install ./cmd/...
+   ```
+   The binaries will be available on your GOPATH.
+    Find it with:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:41]:
```suggestion
1.  The binaries are available on your `GOPATH`. You can find them with this command:
```

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md [hunk 2]
```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
     *   `pg_dump`
     *   `pg_dumpall`
     *   `pg_restore`
-1.  Install `timescaledb-backup`:
+1.  Download `timescaledb-backup`:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:30]:
```suggestion
1.  Download `timescaledb-backup`. If you see an error about 
     not being able to load packages, you can safely ignore them:
```

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md [hunk 3]
```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
...
     ```bash
-    $ go get github.com/timescale/timescaledb-backup/
+    $ go get -d github.com/timescale/timescaledb-backup/
+    # Ignore any can't load packages errors, if any.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:33]:
```suggestion
```

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md [hunk 4]
```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
...
+   ```bash
+   $ cd ~/go/src/github.com/timescale/timescaledb-backup/
+   $ go install ./cmd/...
+   ```
+   The binaries will be available on your GOPATH.
+    Find it with:
+    ```bash
+    $ go env
+    ```
+1. (Optional) Make the binaries available on your path:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:45]:
```suggestion
1. ***Optional:*** Make the binaries available on your path:
```

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md [hunk 5]
```diff
@@ -27,9 +27,26 @@ get`, or download a binary from
...
+     * Debian/Ubuntu:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:46]: Can we add instructions for other OSs, and include them in a `<code>` block?

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md (hunk 3 above)
Comment (snspinn) [reply timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:34]: Thanks for the suggestions. I'm not sure if I'm including them correctly. Can you confirm that the above two suggestions have been accepted by me?

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md (hunk 5 above)
Comment (snspinn) [reply timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:46]: Might take me a week but I can add RHEL/CentOS (7) to this list. Windows and MacOS are beyond me.

**Review by Loquacity — APPROVED** [review]

File: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md (hunk 3 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md:34]: If you go to the `Files Changed` tab in the GH web interface, you can click `Add suggestion to batch` for each suggestion that you want to include. Then you can click `Commit suggestions` at the top of the page when. you're ready. That creates a new commit that you can see in the `Commits` tab.

**Comment by Loquacity** [comment]
Hi @snspinn are you still working on this?

**Comment by Loquacity** [comment]
Appears to be abandoned.

**Pull request closed without merging**